roles/rudder_repository/tasks/debian.yml

```diff
@@ -14,4 +14,14 @@
     state: present
     update_cache: "{{ update_cache }}"
     validate_certs: yes
+  when: repository_url|length == 0
+  become: true
+
+- name: Adding Rudder Repo (from repository_url parameter)
+  apt_repository:
+    repo: "deb {{ repository_url }} main"
+    state: present
+    update_cache: "{{ update_cache }}"
```

Comment:
Why use "update_cache" here, if I read correctly it should be `rudder_update_cache` no ? it is the same everywhere